File: test-schemas-domain-classes/src/main/scala/flatgraph/testdomains/gratefuldead/PropertyNames.java
package flatgraph.testdomains.gratefuldead;

import java.util.HashSet;
import java.util.Set;

public class PropertyNames {


public static final String name = "name";


public static final String performances = "performances";


public static final String songType = "songType";


public static final String weight = "weight";

public static Set<String> ALL = new HashSet<String>() {{
add(name);
add(performances);
add(songType);
add(weight);
}};

}

File: test-schemas-domain-classes-scala3/src/main/scala/testdomains/hierarchical/PropertyNames.scala
package testdomains.hierarchical

import java.util.{HashSet, Set}
import scala.jdk.CollectionConverters.SeqHasAsJava

object PropertyNames {
  val Name: String = "name"

  val All: Set[String] = new HashSet[String](Seq(Name).asJava)
}
